docs/layouts/shortcodes/generated/auto_scaler_configuration.html

```diff
@@ -80,6 +80,18 @@
             <td>Duration</td>
             <td>Expected restart time to be used until the operator can determine it reliably from history.</td>
         </tr>
+        <tr>
+            <td><h5>job.autoscaler.restart.time.tracked.enabled</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Whether to use the actually observed rescaling restart times instead of the fixed 'job.autoscaler.restart.time' configuration. If set to true, the maximum restart duration over a number of samples will be used. The value of 'job.autoscaler.restart.time' will act as an upper bound.</td>
+        </tr>
+        <tr>
+            <td><h5>job.autoscaler.restart.time.tracked.limit</h5></td>
```

Comment:
Sorry for the late comment, we could consider changing this to `job.autoscaler.restart.time-tracking.enabled` and `job.autoscaler.restart.time-tracking.limit` so that we avoid having another config match a prefix (and therefore make it non yaml compliant) 

Not a huge thing but if we want to support yaml configs like in flink we will have to fix it eventually